src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -487,7 +479,7 @@ protected function doSave(array $values, int $lifetime)
         return $failed;
     }
 
-    private function pipeline(\Closure $generator, $redis = null): \Generator
+    private function pipeline(\Closure $generator, object $redis = null): \Generator
```

Comment:
```suggestion
    private function pipeline(\Closure $generator, ?object $redis = null): \Generator
```

Sometimes `?` is used, sometimes not.
What about a linter rule ?v

Reply:
Such a rule would be great yes

Reply:
PHP CS Fixer can be configured to watch over this.